app/test/validation/post.test.js

```diff
@@ -39,12 +46,33 @@ test('check the validation: slug already exists', async()=> {
    expect(result).toContain('The slug already exists');
 
    await postModel.deletePost(post);
-   
+
 })
 
 test('check the validation: title is too short', async()=> {
-    let post = new postModel.PostEntity(null, 'short', 'slug', 'content');
+    let post = new postModel.PostEntity(null, 's', 'slug', 'content');
 
    let result = await validation.validatePost(post);
-   expect(result).toContain('title must be at least 10 characters long');
+   expect(result).toContain('title must be at least 4 characters long');
+})
+
+test('check the validation: slug has forbidden signs', async()=> {
+    let post = new postModel.PostEntity(null, 'swwwwwwww', 'ą ę', 'content');
+
+   let result = await validation.hasForbiddenSigns(post.slug);
+   expect(result).toBe(true);
+})
+
+test('check the validation: slug has not forbidden signs', async()=> {
+    let post = new postModel.PostEntity(null, 'swwwwww', 'wwwwwwww', 'content');
+
+   let result = await validation.hasForbiddenSigns(post.slug);
+   expect(result).toBe(false);
+})
+
+test('check the validation: slug has forbidden signs', async()=> {
+    let post = new postModel.PostEntity(null, 'swwwwww', 2, 'content');
+
+   let result = await validation.hasForbiddenSigns(post.slug);
+   expect(result).toBe(true);
```

Comment:
Chociaż został przekazany zły typ zmiennej, to nie widzę tutaj niedozwolonych znaków. Dlaczego true?